utilities/transactions/write_unprepared_txn.cc

```diff
@@ -263,15 +263,13 @@ Status WriteUnpreparedTxn::FlushWriteBatchToDB(bool prepared) {
 }
 
 Status WriteUnpreparedTxn::FlushWriteBatchToDBInternal(bool prepared) {
+  static std::atomic_ullong autogen_id{0};
   if (name_.empty()) {
     assert(!prepared);
-#ifndef NDEBUG
```

Comment:
Since this will be used only in testing, it might be a bad idea to put back NDEBUG here. And also for autogen_id.